src/lib/pmd/PmdCommandInfo.ts

```diff
@@ -40,6 +41,33 @@ export class Pmd6CommandInfo implements PmdCommandInfo {
 	constructJavaCommandArgsForCpd(fileList: string, minimumTokens: number, language: string): string[] {
 		const classpath = `${PMD6_LIB}/*`;
 		return ['-cp', classpath, HEAP_SIZE, CPD6_MAIN_CLASS, '--filelist', fileList,
-			'--format', 'xml', '--minimum-tokens', String(minimumTokens), '--language', language];
+			'--format', 'xml', '--minimum-tokens', minimumTokens.toString(), '--language', language];
+	}
+}
+
+export class Pmd7CommandInfo implements PmdCommandInfo {
```

Comment:
Not a lot of code here at all. But I had to add quite a bit of testing (because of the poor health of our test code). All this new code is covered 100%.